packages/core/src/api/blockManipulation/blockManipulation.ts

```diff
@@ -141,6 +141,68 @@ export function replaceBlocks<
   blocksToInsert: PartialBlock<BSchema, I, S>[],
   editor: BlockNoteEditor<BSchema, I, S>
 ) {
-  insertBlocks(blocksToInsert, blocksToRemove[0], "before", editor);
-  removeBlocks(blocksToRemove, editor._tiptapEditor);
+  const ttEditor = editor._tiptapEditor;
```

Comment:
this code is now pretty similar to `removeBlocks`. Can / should we reuse the same code?